service.go

```diff
@@ -209,12 +213,23 @@ const configFolder = "config"
 
 // newServiceManager will create a serviceStore out of all the registered Service
 func newServiceManager(c *Server, o *Overlay) *serviceManager {
+	db, err := openDb(c.dbFileName())
+	if err != nil {
+		log.Panic("Failed to create new database: " + err.Error())
+	}
+
 	services := make(map[ServiceID]Service)
-	s := &serviceManager{services, c, network.NewRoutineDispatcher()}
+	s := &serviceManager{services, c, db, network.NewRoutineDispatcher()}
 	ids := ServiceFactory.registeredServiceIDs()
 	for _, id := range ids {
 		name := ServiceFactory.Name(id)
 		log.Lvl3("Starting service", name)
+
+		err = createBucketForService(s.db, name)
+		if err != nil {
+			log.Panic("Failed to create bucket: " + err.Error())
```

Comment:
Panic does not seem like the right choice.

Reply:
According to the documentation, `tx.CreateBucketIfNotExists` would only fail if the bucket name is empty or it is too long. As far as I understand, it's not like a user input and the programmer should have full control of the service name. If it is empty or too long, then it's a programmer error rather than an error that can be handled, hence we panic.

Reply:
ok, got it.